gating/thaw/haproxycheck.yml

```diff
@@ -0,0 +1,37 @@
+- name: Check HAProxy status after reconfiguration
+  hosts: localhost
+  tasks:
+  - name: Check state of haproxy backends
+    shell: |
+      set -xeu
+      # Check haproxy backends excluding elasticsearch (Future work required
+      # to get elasticsearch working post thaw).
+      # Haproxy prints the stats itteration so if the line is "1" its ignored
+      # Lines where the first fied is hash are comments and are ignored
```

Comment:
Nit: typos on itteration, fied, may want to add an explicit TODO on the elasticsearch piece.

Reply:
Done.

Reply:
Added https://rpc-openstack.atlassian.net/browse/RE-1192 for follow up.